operations/helm/charts/mimir-distributed/templates/ingester/ingester-statefulset.yaml

```diff
@@ -15,6 +15,15 @@ metadata:
 spec:
   podManagementPolicy: {{ .Values.ingester.podManagementPolicy }}
   replicas: {{ $rolloutZone.replicas }}
+  {{- if and (semverCompare ">= 1.27.x" (include "mimir.kubeVersion" .)) (.Values.ingester.persistentVolume.enableStatefulSetAutoDeletePVC)  }}
+  {{/*
+    Data on the read nodes is easy to replace, so we want to always delete PVCs to make
+    operation easier, and will rely on re-fetching data when needed.
```

Comment:
this comment is misleading. The ingesters also contain recently ingested data which is not stored anywhere else. I think we should remove it